### PR TITLE
v1.10.0 pre release changes

### PR DESCRIPTION
**Pull request by agoallikmaa**

### CODE REVIEW

**Comment by laurit** [comment]
next release should be `1.10.0` not `1.9.2`

**Comment by agoallikmaa** [comment]
Fixed, 1.10.0 now.

File: CHANGELOG.md [hunk 1]
```diff
@@ -101,7 +118,7 @@ and this repository adheres to [Semantic Versioning](https://semver.org/spec/v2.
 
 ### General
 
-- OpenTelemetry Instrumentation for Java has been updated to version 1.9.2.
+- OpenTelemetry Instrumentation for Java has been updated to version 1.10.0.
```
Comment (mateuszrzeszutek) [inline CHANGELOG.md:121]: Please revert this line.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,23 @@ and this repository adheres to [Semantic Versioning](https://semver.org/spec/v2.
 
 ## Unreleased
 
+## v1.10.0 - 2022-04-22
+
+### General
+
+- OpenTelemetry Instrumentation for Java has been updated to version 1.13.0.
```
Comment (mateuszrzeszutek) [inline CHANGELOG.md:15]:
```suggestion
- OpenTelemetry Java SDK and OpenTelemetry Instrumentation for Java dependencies have been updated to version 1.13.0.
```

File: README.md [hunk 3]
```diff
@@ -164,11 +159,11 @@ To extend the instrumentation with the OpenTelemetry Instrumentation for Java,
...
 <!-- IMPORTANT: do not change comments or break those lines below -->
-The Splunk Distribution of OpenTelemetry Java version <!--SPLUNK_VERSION-->1.9.1<!--SPLUNK_VERSION--> is compatible
+The Splunk Distribution of OpenTelemetry Java version <!--SPLUNK_VERSION-->1.10.0<!--SPLUNK_VERSION--> is compatible
 with:
 
-* OpenTelemetry API version <!--OTEL_VERSION-->1.12.0<!--OTEL_VERSION-->
-* OpenTelemetry Instrumentation for Java version <!--OTEL_INSTRUMENTATION_VERSION-->1.12.1<!--OTEL_INSTRUMENTATION_VERSION-->
+* OpenTelemetry API version <!--OTEL_VERSION-->1.13.0<!--OTEL_VERSION-->
+* OpenTelemetry Instrumentation for Java version <!--OTEL_INSTRUMENTATION_VERSION-->1.13.0<!--OTEL_INSTRUMENTATION_VERSION-->
 * Micrometer version 1.8.4
```
Comment (mateuszrzeszutek) [inline README.md:167]:
```suggestion
* Micrometer version 1.8.5
```

**Review by mateuszrzeszutek — APPROVED** [review]

**Review by laurit — APPROVED** [review]

**Pull request merged**